tools/cldr-apps/js/src/views/DashboardWidget.vue

```diff
@@ -255,22 +255,23 @@ export default {
 
     fetchData() {
       if (!cldrStatus.getSurveyUser()) {
-        this.fetchErr = "Please log in to see the Dashboard.";
+        this.fetchErr.value = "Please log in to see the Dashboard.";
         return;
       }
       this.locale = cldrStatus.getCurrentLocale();
       this.level = cldrCoverage.effectiveName(this.locale);
       if (!this.locale || !this.level) {
-        this.fetchErr = "Please choose a locale and a coverage level first.";
+        this.fetchErr.value =
+          "Please choose a locale and a coverage level first.";
```

Comment:
Does this work right in conjunction with `<p v-if="fetchErr" class="st-sad">{{ fetchErr }}</p>`?

In other words, will `{{ fetchErr }}` automatically show fetchErr.value?